ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md

```diff
@@ -2528,6 +2528,9 @@ router_bgp:
       ebgp_multihop: < integer >
       next_hop_self: < true | false >
       password: "< encrypted_password >"
+      default_originate:
+        always: < true | false >
```

Comment:
I have to withdraw my approval. This data model will end up with an empty key in the event of no route-map and not always. That will not work. So we need to add an `enabled: < true | false >` here.

Reply:
Understand. To comply with the old model defined in other sections of bgp, I went with this data model. I will make changes though.